.github/workflows/actions/patch_image_and_check_diff/action.yml

```diff
@@ -72,6 +72,13 @@ runs:
         
         sleep 10
 
+    - name: Log CW Agent Operator spec container args before patching
+      id: default-cw-agent-operator-image
```

Comment:
Remove or update this id for this step, as it is already being used

Reply:
Fixed